libs/island-ui/core/src/lib/InfoCardGrid/InfoCardGrid.tsx

```diff
@@ -45,6 +46,7 @@ export const InfoCardGrid = ({
   cardsBorder,
   variant,
   columns,
+  notFoundText,
 }: Props) => {
   const [isMobile, setIsMobile] = useState(false)
```

Comment:
Is this state necessary, could you also just do `const isMobile = width < theme.breakpoints.sm`?

Reply:
I'm talking about `const [isMobile, setIsMobile] = useState(false)`

Reply:
I wonder if during the initial load on mobile the UI will jump around because useEffect runs after the component mounts

Reply:
hm yeah i'll simplify